lib/replaceRef.js

```diff
@@ -6,38 +6,40 @@ var fs = require('fs');
 
 module.exports = replaceRef;
 
-// TODO: allow this to be command-line configurable
-var ignorableTypes = ['gltfid.schema.json', 'gltfchildofrootproperty.schema.json', 'gltfproperty.schema.json'];
```

Comment:
The README already has an example of how to use it.  I accidentally had added that in the previous commit due to some confusion with which featureset was in which of my branches at the time.

I originally wasn't going to have this feature at all (hence just the TODO) because I thought Wetzel was only being used by glTF, but in the month since I originally implemented the type linking, I saw (from the PR's) that people were trying to use it for other schemas.  Given that, I don't know if it makes sense to hard-code the glTF set in there, provided that the glTF set is clearly identified in the documentation (which it is).

If there's something in the documentation (or help) that should be made more clear, let me know and I'll be happy to do so.  I'm also open to an alternative method of doing this...this was the best idea I could come up with.